sqlanalytics/resource_sql_global_config.go

```diff
@@ -0,0 +1,117 @@
+package sqlanalytics
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type SQLEndpointConfPair struct {
```

Comment:
We already have a similar entity for tags in sql endpoint. Though dunno if it's good idea to reuse it